tests/integration/conformance/testdata/networking/traffic/test.yaml

```diff
@@ -0,0 +1,5 @@
+isolated: true
+environments:
+  - kube
```

Comment:
is there any reason this cannot be native? I tried getting a similar test (trafficshifting) running on native without so success so just wondering if you tried it

Reply:
Didn't try. I care only about kube and didn't want to end up digging into details of other envs too much.